tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/ASM/AST/DocumentNode.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="DocumentNode.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate.ASM.AST
+{
+    /// <summary>
+    /// HotChocolate.Language.DocumentNode interface for ducktyping
+    /// https://github.com/ChilliCream/graphql-platform/blob/423e8a3285f4d20291bc78ce09fed23a091a01d0/src/HotChocolate/Language/src/Language.SyntaxTree/DocumentNode.cs
+    /// </summary>
+    [DuckCopy]
+    internal struct DocumentNode
+    {
+        public IEnumerable<object> Definitions;
```

Comment:
This won't work unfortunately, you'll need to duck type it as the non-generic `IEnumerable` instead
```suggestion
        public IEnumerable Definitions;
```

Reply:
fixed see 8e67bc2d3bf40bedb468c49a28e474862c23c105